site/en/docs/extensions/migrating/known-issues/index.md

```diff
@@ -15,69 +15,48 @@ tags:
 
 {% endAside %}
 
-This page lists major known issues that affect developers as they migrate to Manifest V3. Known issues are divided into two primary groups:
+Recently, we announced changes to the Manifest V2 (MV2) deprecation timeline, and while we remain firmly committed to Manifest V3 (MV3) we acknowledge there is more work to do on our part including: 
```

Comment:
```suggestion
Recently, we announced changes to the Manifest V2 (MV2) deprecation timeline, and while we remain firmly committed to Manifest V3 (MV3) we acknowledge there is more work to do on our part.
```

Optional, but the "including" followed by bullets for "we will do <x>" sounds a little awkward.